python/rikai/contrib/s3.py

```diff
@@ -0,0 +1,49 @@
+#  Copyright (c) 2021 Rikai Authors
```

Comment:
this could be put into main rikai repo instead of `contrib` tho.

Reply:
yeah, was just putting it here til we have a "rikai blessed" s3 interface for all the things.